src/java/org/apache/cassandra/service/CassandraDaemon.java

```diff
@@ -453,12 +454,14 @@ public void startNativeTransport()
             throw new IllegalStateException("native transport should be set up before it can be started");
 
         nativeServer.start();
-        logger.info("Native server running on {}", new InetSocketAddress(DatabaseDescriptor.getRpcAddress(), DatabaseDescriptor.getNativeTransportPort()));
+        InetSocketAddress nativeAddress = new InetSocketAddress(DatabaseDescriptor.getRpcAddress(), DatabaseDescriptor.getNativeTransportPort());
+        logger.info("Native server running on {}", SafeArg.of("nativeAddress", nativeAddress));
```

Comment:
Think it might be better to split the parameters here actually. It doesn't matter much here, but both this and the log line below can both use `address` and `port` for param name consistency, as the message already differentiates native vs. thrift.